.github/workflows/peribolos.yaml

```diff
@@ -0,0 +1,20 @@
+name: Apply Organization Membership
+on:
+  workflow_dispatch: {}
+  push:
+    branches:
+      - 'main'
+    paths:
+      - "config/opendatahub-io/org.yaml"
+  pull_request_review:
+jobs:
+  peribolos:
+    runs-on: ubuntu-latest
+    container:
+      image: gcr.io/k8s-prow/peribolos
+    steps:
+      - uses: actions/checkout@v3
+      - name: Apply ODH organization memebership
+        run: |
+          echo ${{ secrets.ORG_MANAGEMENT_TOKEN }} > /.github_token.txt
+          peribolos --github-token-path /.github_token.txt --config-path "config/opendatahub-io/org.yaml"  --fix-org --fix-org-members --fix-teams --fix-team-members --fix-team-repos --fix-repos --confirm
```

Comment:
How do you plan to pass the secret to a file?

Reply:
@LaVLaS the intent is to register the token as an org secret (there already is one, but I think it might need to be updated) and then that secret gets pulled from the env var and written to a file (line 19)

Reply:
@accorvin Ignore this. I completely overlooked the previous line where you dumped it to a file.